regimes/pt/README.md

```diff
@@ -21,9 +21,43 @@ Portugal doesn't have an e-invoicing format per se. Tax information is reported
 | ND | Nota de débito | `credit-note` | |
 | NC | Nota de crédito | `debit-note` | |
 
+### `TaxCountryRegion` (País ou região do imposto)
+
```

Comment:
I think we need more details here for how these fields will be used, I tried to do the same for Italy already. The "GOBL Tax Identity Zone" also feels a bit out of place, it looks like we made up a number. These are actually the zone codes from ISO 3166-2:PT (according to the comment in the code: https://github.com/invopop/gobl/blob/main/regimes/pt/pt.go#L23)

Reply:
Local codes descriptions added in f055a5bde1a0e2f7dcdff3ab3ffaea1bd95e54af. Thank you for the feedback!